lib/quicly.c

```diff
@@ -1505,10 +1513,11 @@ int quicly_encode_transport_parameter_list(ptls_buffer_t *buf, int is_client, co
     /* if requested, add a greasing TP of 1 MTU size so that CH spans across multiple packets */
     if (expand) {
         PUSH_TP(buf, 31 * 100 + 27, {
-            if ((ret = ptls_buffer_reserve(buf, QUICLY_MAX_PACKET_SIZE)) != 0)
+            static const size_t bigger_than_mtu = 1600;
```

Comment:
Why does this greasing size have to be bigger?

Reply:
The intent of the `expand` flag is to test quantum readiness of the protocol stack, where ClientHello does not fit into one packet. The Q bit of the QUIC interop matrix.